visualization/app/codeCharta/state/store.service.spec.ts

```diff
@@ -118,24 +116,5 @@ describe("StoreService", () => {
 
 			expect(storeService.getState().appSettings.isLoadingMap).toBeFalsy()
 		})
-
-		describe("connection to action subject", () => {
```

Comment:
Moved the logic and test into plain store, so that effects works also in tests, which don't inject AngularJS' store.service